frontend/src/hooks/use-network-context.ts

```diff
@@ -0,0 +1,10 @@
+import { useContext } from "react";
+import { NetworkContext } from "../context/NetworkContext";
+
+export const useNetworkContext = (): NetworkContext => {
```

Comment:
It could be that name `useSureNetworkContext` would be semantically better (or some other prefix explicitly stating that context should always be here)

Reply:
I feel the current name + strict type is already quite explicit. I feel I would be more confused trying to parse `useSureNetworkContext`